infinity-app/app.R

```diff
@@ -128,12 +171,24 @@ ui <-
       selectizeInput(
         inputId = "taxa_family",
         label = "Choose a family: (you can also select All, but it's slow so be patient)",
-        choices = "Myrtaceae",
-        selected = "Myrtaceae",
+        choices = "My",
+        selected = "All",
         options = list(maxOptions = 300L)
       )
     ),
-    
+    selectInput(
```

Comment:
Irrelevant comment: I had this as a slider bar, I think it would look better in this context! Same for the butters

Reply:
@tmesaglio likes the precision of the discrete choices, i'll let you guys discuss